code-challenges.test.js

```diff
@@ -20,19 +20,54 @@ const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+// a describe method that lists the name of the function OR naming of the particular test.
+describe(noPurpleNoChartreuse, () => {
+
+    it("takes in a array and returns a array with no purple or chartreuse", () => {
+      expect(noPurpleNoChartreuse(colors1)).toEqual(["yellow", "blue", "pink", "green"])
+      expect(noPurpleNoChartreuse(colors2)).toEqual(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"])
+    })
+  })
 
 // b) Create the function that makes the test pass.
 
+// filter(function noPurpleNoChartreuse(element){/*"purple", "chartreuse"*/})
+
```

Comment:
Please review the question again. 

1. The expect statement on the test is missing some syntax.

https://jestjs.io/docs/expect#expectarraycontainingarray

2. The new array needs to be shuffled.